### PR TITLE
Improved translations

### PR DESCRIPTION
**Pull request by marcosgdf**
Added missing translations and an improvement in DONOTSEARCH_ANYWHERE constants (moved info into tooltip)

### CODE REVIEW

File: htdocs/langs/en_US/banks.lang [hunk 1]
```diff
@@ -133,6 +133,7 @@ CashBudget=Cash budget
 PlannedTransactions=Planned transactions
 Graph=Graphics
 ExportDataset_banque_1=Bank transactions and account statement
+ExportDataset_banque_1=Deposit slip
```
Comment (rdoursenaud) [inline htdocs/langs/en_US/banks.lang]: Looks like you introduced a duplicate.

Same goes for es_ES and fr_FR

Comment (marcosgdf) [inline htdocs/langs/en_US/banks.lang]: Fixed! Thanks

**Comment by simnandez** [comment]
'es_ES' and 'fr_FR' must be translate with transifex, only 'en_US' will be translate with git

**Comment by marcosgdf** [comment]
Good to know. Removed!

**Pull request merged**